client-app/src/desktop/tabs/other/AppNotificationsPanel.js

```diff
@@ -77,7 +77,7 @@ const idleServiceTile = hoistCmp.factory(
                     className: 'tb-idle__button',
                     icon: Icon.userClock(),
                     text: 'Enter Sleep Mode',
-                    onClick: () => XH.idleService.suspendApp()
+                    onClick: () => XH.suspendApp({reason: 'IDLE'})
```

Comment:
unrelated to model change yeah?

Reply:
yep -- probably should have fixed directly against develop